### PR TITLE
Breaking change: Remove expand(_forward_) and expand(_reverse_).

### PR DESCRIPTION
**Pull request by martinmr**
This change is [Reviewable](https://reviewable.io/reviews/dgraph-io/dgraph/4119)

### CODE REVIEW

File: wiki/content/query-language/index.md [hunk 1]
```diff
@@ -1800,10 +1800,6 @@ There are four ways to use the `expand` function.
 * If `_all_` is passed as an argument to `expand()`, all the predicates for each
```
Comment (campoy) [inline wiki/content/query-language/index.md]: Since we're at it, this statement is not accurate.

File: query/query.go [hunk 2]
```diff
@@ -1824,7 +1824,7 @@ func expandSubgraph(ctx context.Context, sg *SubGraph) ([]*SubGraph, error) {
 		}
 
 		switch child.Params.Expand {
-		// It could be expand(_all_), expand(_forward_), expand(_reverse_) or expand(val(x)).
+		// It could be expand(_all_) or expand(val(x)).
```
Comment (campoy) [inline query/query.go]: What's val(x)? I have never any example like this

File: wiki/content/query-language/index.md (hunk 1 above)
Comment (campoy) [inline wiki/content/query-language/index.md]: Since we're at it, this statement is not accurate.

File: query/query.go (hunk 2 above)
Comment (MichelDiz) [reply query/query.go]: This was a variable that gets predicates values from the old `_predicate_` func. To expand a query with those preds. It was like "expand only the preds found in the `_predicate_` func".

Comment (martinmr) [reply query/query.go]: Pasted from my reply in Reviewable: "It takes a variable (a list of string values) and uses those values as the names of the predicates to expand. We should discuss whether to keep it but I left it untouched for now."

File: gql/parser.go [hunk 3]
```diff
@@ -2802,10 +2802,6 @@ func godeep(it *lex.ItemIterator, gq *GraphQuery) error {
 					child.Expand = child.NeedsVar[len(child.NeedsVar)-1].Name
 				case "_all_":
 					child.Expand = "_all_"
-				case "_forward_":
-					child.Expand = "_forward_"
-				case "_reverse_":
-					child.Expand = "_reverse_"
```
[the remark this answers is not in the document]
Comment (martinmr) [reply gql/parser.go]: Added error message informing users about the deprecation of these features.

**Pull request merged**